src/Parser/Druid/Restoration/Modules/Items/GarothiFeedbackConduit.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { formatThousands, formatPercentage, formatNumber } from 'common/format';
+
+import CoreGarothiFeedbackConduit from 'Parser/Core/Modules/Items/GarothiFeedbackConduit';
+import STAT from "Parser/Core/Modules/Features/STAT";
+import StatTracker from "Parser/Core/Modules/StatTracker";
+import HealingDone from 'Parser/Core/Modules/HealingDone';
+import ITEMS from 'common/ITEMS';
+import SPELLS from "common/SPELLS";
+import StatWeights from '../Features/StatWeights';
+
+class GarothiFeedbackConduit extends CoreGarothiFeedbackConduit {
+  static dependencies = {
+    ...CoreGarothiFeedbackConduit.dependencies,
+    statWeights: StatWeights,
+    statTracker: StatTracker,
+    healingDone: HealingDone,
+  };
+
+  totalIntellect = 0;
+
+  on_initialized() {
+    super.on_initialized();
+    this.totalIntellect = this.statTracker.startingIntellectRating;
+  }
+
+  on_toPlayer_applybuff(event) {
+    const spellId = event.ability.guid;
+    if(!event.prepull) {
+      return;
+    }
+    if(spellId === SPELLS.POTION_OF_PROLONGED_POWER.id) {
+      this.totalIntellect -= 2500;
+    }
+    if(spellId === SPELLS.DEFILED_AUGMENT_RUNE.id) {
+      this.totalIntellect -= 325;
+    }
+    if(spellId === SPELLS.FLASK_OF_THE_WHISPERED_PACT.id) {
+      this.totalIntellect -= 1300;
+    }
+  }
+
+  item() {
+    const avgHaste =
+      this.totalProccValue.reduce((acc, proc) => acc + (proc[0] * proc[1]), 0) / this.owner.fightDuration;
+    //TODO: Add the healing from each haste rating in the future:
+    const hpmWeight = this.statWeights._getGain(STAT.HASTE_HPM) / (this.statWeights.totalOneInt || 1);
+    const throughput = (hpmWeight * avgHaste) / this.totalIntellect;
```

Comment:
Ok, I've had the time to more closely look over this calculation.

Psuedocode -
Your throughput calculation: `(hasteHealingFromOne / intHealingFromOne) * avgHaste / "totalInt"`
Honestly the more I look at it the less sense it makes. The units aren't even right, see: `(healing / healing) * scalar / statAmount = 1 / statAmount` You want units of healing, and ended up with 1 / statAmount. All of this is even ignoring that 'totalInt' doesn't properly reflect procs. It's a miracle this produced a value that looked kinda correct.

My suggested throughput calculation: `hasteHealingFromOne * avgHaste`
This just takes the healing gained from one haste, and multiplies by haste gain. Units: `healing * scalar = healing`.

Reply:
Ohh, your calculation is supposed to be percentage of total throughput, not raw throughput. The calculation is still wrong, but I better understand how it got a number that looked right.

Reply:
The reasoning for having a throughput number is for users more easily compare with other trinkets. 
For example- Let's say we have another trinket with a healing procc, it has the same amount of INT as Garothis but the healing procc did 3% healing. How do you as a user decide if 3% healing is better than 1.9k haste rating? 
* Normally (as in Tortys spreadsheet too afaik) we translate 3% healing to INT by taking 0.03 * totalIntUnbuffed. 
* So for a character with 70k INT, 3% healing would be worth 2100 INT. 
* Assuming haste is weighted at 1.3, the 1.9k haste rating gained by Garothis would be worth 2.47k int. Thus making Garothis stronger.
* Using the variable ``hasteHealingFromOne`` that would yield roughly 2.25% in this example, showing Garothis way worse than the healing proc trinket.

Now I understand that your ``hasteHealingFromOne`` *should* yield the correct healing value per rating, but it seems odd that it differs by 2/3 of the expected values comparing to the values in the spreadsheets. This may cause some confusion for users trying to verify/calculate the worth of the trinket I think.

Reply:
On another hand, I guess the ``hasteHealingFromOne`` should yield the most accurate values, since it's a haste procc, it may procc during times of downtime and maybe that's why it's showing a lot less value than expected?

Reply:
To extend, i'd like to take the approach of using ``xxxHealingFromOne`` variables from constant stat buffs such as stat sticks and see if they yield a more close to the expected values.

Reply:
Ah, I understand how you came up with this calculation now. The calculation's error is the assumption that `gainedInt / totalUnbuffedInt = percentGainedThroughput`. I would expect the resulting number to overstate things significantly. Using the unbuffed int value is just wrong, especially considering how many int buffs there are. Once flask, food, rune, and concordance are factored in, the player's actual int will always be higher than the unbuffed int, and it's the actual int that matters when trying to calculate percent throughput gain. Also, the set of spells that scale from int is different from the set that scale with haste, that too could factor in. Also, the hasteHealingFromOne value ignores Tranq, while your the way you've done your calculation effectively unignores Tranq; that's another explanation for your value being higher.

The purpose of the StatValues calculator was to remove a lot of the guesswork inherent in Torty's spreadsheet. Your calculation just adds that guesswork back in.

Reply:
Fair enough ^^ I'll push the changes tomorrow